arches_project/core/arches/resources.py

```diff
@@ -145,13 +148,13 @@ def close_dialog(dialog):
 
         if arches_selected_resource:
             selectedLayerIndex = dlg.editResSelectFeatures.currentIndex()
-            selectedLayer = self.layers[selectedLayerIndex]
+            selectedLayer = self.archesproject.layers[selectedLayerIndex]
 
             geom_convert = Geometries(selectedLayer)
             geomcoll, geometry_type_dict = geom_convert.geometry_conversion()
 
             # Get nodegroup from graph
-            for graph in self.arches_graphs_list:
+            for graph in self.archesproject.layers.archesproject.arches_graphs_list:
```

Comment:
Does this mean to be?
```suggestion
            for graph in self.archesproject.arches_graphs_list:
```